mapbox/dr-ui
# Create `resultsOnly`, `segmentTrackEvent`, and `overrideSearchTerm` props for `Search` component

This PR adds three new properties to Search:
- `resultOnly` - If true, only show results from search
- `segmentTrackEvent` - Set the segment track event, default is (`Searched docs`)
- `overrideSearchTerm` - If `resultsOnly: true` set `overrideSearchTerm` to set the search term

These properties will allow the Support team to extend the Search component to power suggestions in the new Support contact form: https://github.com/mapbox/support-center/pull/19

## How to test

1. Run `npm start`.
1. Open /Search test cases page.
1. Scroll to `Set resultsOnly to true.` test case.
1. Select an item from the test cases Select element.
1. Results for item should appear below the select.

## QA checklist




Open the test cases app locally on:
